qa/tasks/cephfs/test_mirroring.py

```diff
@@ -567,10 +567,14 @@ def test_cephfs_mirror_stats(self):
                                "client.mirror_remote@ceph", '/d0', 'snap0', 1)
         self.verify_snapshot('d0', 'snap0')
 
-        # check snaps_synced
+        # check perf counters
         res = self.mirror_daemon_command(f'counter dump for fs: {self.primary_fs_name}', 'counter', 'dump')
         second = res[TestMirroring.PERF_COUNTER_KEY_NAME_CEPHFS_MIRROR_PEER][0]
         self.assertGreater(second["counters"]["snaps_synced"], first["counters"]["snaps_synced"])
+        self.assertGreater(second["counters"]["last_synced_start"], first["counters"]["last_synced_start"])
+        self.assertGreater(second["counters"]["last_synced_end"], second["counters"]["last_synced_start"])
+        self.assertGreater(second["counters"]["last_synced_duration"], 0)
+        self.assertGreater(second["counters"]["last_synced_bytes"], 0)
```

Comment:
Ideally, you can compare this to the file size (or at least least greater than or equal to that considering some metadata amounted).

Reply:
I'd like to do such kind of check, but `create_n_files` doesn't return any size info. We just have `50`. Is that 50 bytes total?

Reply:
Oh I meant write some data to some files and then expect the perf counter to account that size.

Reply:
Got it, fixing it.